pages/vi/profiles/EdwinKaburu.md

```diff
@@ -0,0 +1,15 @@
+# Hello Everyone
+I am Edwin Kaburu. I am from Kent, WA (TimeZone : Pacific Time). I am currently using Windows 10
+I am currently pursing my Bachelors at Renton Technical College. ![alt text](https://www.rtc.edu/sites/default/files/vital-rtccampus-1000x500.jpg)
+I have a Application Developer Certificate from Renton Technical College. The technical skills that I have come to learn over the past few years include
+ - C#
```

Comment:
@EdwinKaburu this is not creating bullets, please also check githack link for output